packages/tradershub/src/components/RegulationSwitcher/RegulationSwitcherMobile.tsx

```diff
@@ -13,7 +13,7 @@ const RegulationSwitcherMobile = () => {
     const activeRegulation = uiState.regulation;
 
     return (
-        <div className='flex items-center gap-8'>
+        <div className='flex items-center gap-8 w-[49%] lg:w-auto justify-end '>
```

Comment:
Why 49% ?

Reply:
@mitra-deriv two elements can have 50% each if we need them in a row. I didn't get using 49%

Reply:
Correct I tried 50% first it wasn't getting to center somehow so I used 49% I double checked now I think I may had some CSS changes  while I was trying so it didn't become center,  now with 50% it is fine also I will change it , thanks .